### PR TITLE
Added select_many() to exposed API

### LINKED ISSUE #68: Expand exposed API with select_many() function
User should be able to select many items by theirs IDs:

``` javascript
var ids = ["Chrome","Safari"];
$scope.api.select_many(ids);
```

This way the api gets more flexible.

### PR DESCRIPTION
**Pull request by panayotkulchev**
Fixes #68

User should be able to select many items by theirs IDs:

``` javascript
var ids = ["Chrome","Safari"];
$scope.api.select_many(ids);
```

This way the api gets more flexible.

In my case I have different user roles like: `admin` , 'superAdmin', 'user' . Each has different set of permissions. 
So when new user has f.e. `admin` and `user` roles, their permissions are merged and call select_many(merged_ids)

Would be good if later `deselect()`, `deselect_many()` are implemented. 
Also to be configurable: when item is selected, whether parent to be checked or not.

### CODE REVIEW

**Comment by alexandernst** [comment]
Hi, thank you for the PR. Can you please fix the indent levels (look at `https://github.com/alexandernst/angular-multi-select/pull/69/files` to check the difference in the indentation level).
Also, the unit test you added isn't passing, can you fix that too?

`Also to be configurable: when item is selected, whether parent to be checked or not.`

This was implemented here: https://github.com/alexandernst/angular-multi-select/pull/57

File: spec/helpers/multi_selected.data.js [hunk 1]
```diff
@@ -22,6 +22,7 @@ var multi_test_data = [
 				name: "Open Source",
 				sub: [
 					{
+            id: 41,
```
Comment (panayotkulchev) [inline spec/helpers/multi_selected.data.js]: If you have indentation level issue here - it's my IDE or GHub issue. What I have is:

idents

And all is as should be.

Comment (alexandernst) [inline spec/helpers/multi_selected.data.js]: Tell your IDE to show TABs and SPACEs and you'll see the difference.

**Comment by panayotkulchev** [comment]
It was the IDE. When press tab it adds couple empty spaces or smth like this.

**Comment by alexandernst** [comment]
Yes, IDEs usually add spaces (unless you tell them to add tabs). Can you fix the indent of the other two files too?

**Comment by panayotkulchev** [comment]
done

File: src/angular-multi-select.js [hunk 2]
```diff
@@ -135,6 +135,16 @@ angular_multi_select.directive('angularMultiSelect',
...
+            $timeout(function() {
+              angular.forEach(ids, function (id) {
+                var item = $scope._getItemById(id);
+                if (item !== null && !$scope._isChecked(item)) {
+                  $scope.clickItem(item, true);
+                }
+              }, 0);
+            });
+          },
 					reset: function() {
```
Comment (alexandernst) [inline src/angular-multi-select.js:148]: You're still missing the indent of this one.

**Comment by panayotkulchev** [comment]
fixed

**Comment by panayotkulchev** [comment]
When do you intend to make new release ?

**Comment by alexandernst** [comment]
@panayotkulchev Today if I manage to get 15 min of spare time ;)

**Comment by alexandernst** [comment]
@panayotkulchev Can you add a couple of lines to the docs about this feature? (branch gh-pages)

**Comment by panayotkulchev** [comment]
Docs added. Hope it is enough.

**Comment by alexandernst** [comment]
Perfect :+1:

**Comment by alexandernst** [comment]
@panayotkulchev 5.5.4 was just released. ;)

**Pull request merged**